### ISSUE TITLE
Save to firestore after upload photos is not working?

### ISSUE THREAD

**Issue by dirathea** [prescottprue/react-redux-firebase#600]
**What is the current behavior?**
I'm using react-redux-firebase@next with firestore, and tried to use firebase.uploadFiles. The file is uploaded but the record is not saved on firebase even the configuration is already set up :

config
```
const rrfConfig = {
  userProfile: 'users',
  useFirestoreForProfile: true,
  useFirestoreForStorageMeta: true,
  enableLogging: false,
};

export const rrfProps = {
  firebase,
  config: rrfConfig,
  dispatch: store.dispatch,
  createFirestoreInstance,
};
```

implementation
```

const uploadPath = 'images';
class UploadField extends Component {
  onDrop = files => {
    this.props.firebase
      .uploadFiles(uploadPath, files, uploadPath)
      .then(result => {
        console.log(result);
      });
  };

  render() {
    return (
      <Dropzone onDrop={this.onDrop}>
      </Dropzone>
    );
  }
}

export default compose(
  withFirebase
)(UploadField);
```

**What is the expected behavior?**

The file uploaded and record is added

Thanks.

**Comment by dirathea** [comment]
Ah turns out, the config must include `firebase.database` [in this line](https://github.com/prescottprue/react-redux-firebase/blob/next/src/actions/storage.js#L67) even I only use firestore and state `useFirestoreForStorageMeta` on config.

Am I doing it wrong or it should be configure without firebase database if the user only use firestore?

**Comment by prescottprue** [comment]
Fixed in both v2.2.6 and v3.0.0-alpha.4. Thanks for the PR
